app/static/ie-status.json

```diff
@@ -4228,6 +4228,33 @@
     "uservoiceid": null
   },
   {
+    "name": "Backdrop filter",
+    "category": "CSS",
+    "link": "http://dev.w3.org/fxtf/filters-2/#BackdropFilterProperty",
+    "summary": "Applies a CSS filter effect to the content behind an element. This is not the element's background but the content that would be drawn behind it.",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text":"Not currently planned",
```

Comment:
This is "Under Consideration" for us.  @dstorey can you check with the composition/rendering team on a priority level for this one and get it added?